src/analytics/Properties.tsx

```diff
@@ -1229,9 +1229,6 @@ export interface WalletConnect2Properties {
   dappUrl: string
   dappDescription: string
   dappIcon: string
-  permissionsBlockchains: string[]
-  permissionsJsonrpcMethods: string[]
-  permissionsNotificationsTypes: string[]
```

Comment:
Could we mention as a comment that we should have `eip155Events`,  `eip155Chains` and `eip155Methods` added dynamically?

Reply:
yup, added a comment!